invenio_app_rdm/theme/assets/semantic-ui/js/invenio_app_rdm/landing_page/RecordCitationField.js

```diff
@@ -106,20 +107,20 @@ export class RecordCitationField extends Component {
               <List celled horizontal className="separated-list">
                 {styles.map((style) => {
                   return (
-                    <List.Item>
+                    <List.Item key={style[0]}>
                       <a
-                        onClick={() => this.getCitation(record, style[0])}
-                        className={
-                          loading
-                            ? "citation-style-link loading"
-                            : "citation-style-link"
-                        }
+                        href="#"
```

Comment:
I think this can be removed now if it doesn't break the fix :)

Reply:
yup 🔥